res/values/strings.xml

```diff
@@ -36,4 +36,16 @@
     <string name="swipe_gestures_title">Swipe to navigate your phone</string>
     <string name="swipe_gestures_desc">Learn gestures to go home, go back and switch between apps</string>
     <string name="try_it">Try it</string>
+    <string name="oem_unlocked_warning">The boot loader is unlocked and software integrity can not be guaranteed. Any data stored on this device may be available to attackers. Do not store any sensitive data on this device.</string>
+    <string name="visit_this_link_on_another_device">Visit this link on another device:</string>
+    <string name="oem_lock_url" translatable="false">g.co/ABH</string>
+    <string name="oem_unlocked_device_setup_confirmation">Boot loader is unlocked which renders the device insecure as displayed in the warning. Do you still want to continue setting up?</string>
```

Comment:
This text should refer the user back to the installation steps on grapheneos.org

Reply:
yes this is pending actually, particularly this is why I've still kept the PR as draft.
there were different opinions regarding this on whether to just refer to existing installation steps or have a dedicated page and shorter url.

Reply:
resolved now